### PR TITLE
added ibc docs and new favicon

### PR DESCRIPTION
**Pull request by bengtlofgren**

### CODE REVIEW

File: documentation/docs/src/user-guide/ibc.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
+# Using IBC with Namada
+
+This document describes using the inter-blockchain communication (IBC) protocol with Namada. This documentation covers being able to create connections through IBC as well as setting up local fractal instances of Namada for testing purposes.
```
Comment (cwgoes) [inline documentation/docs/src/user-guide/ibc.md:3]:
```suggestion
This document describes using the inter-blockchain communication (IBC) protocol with Namada. This documentation covers being able to create connections through IBC as well as setting up local instances of Namada for testing purposes.
```

Comment (cwgoes) [reply documentation/docs/src/user-guide/ibc.md:3]: we should avoid using this term too much until we've defined it more precisely

**Review by cwgoes — APPROVED** [review]
utACK

**Review by brentstone — APPROVED** [review]
LGTM!

**Pull request merged**